integrationTests/package.json

```diff
@@ -0,0 +1,95 @@
+{
+  "name": "commerce-sdk-integration-test",
+  "version": "1.4.1-beta.0",
+  "description": "Salesforce Commerce Cloud SDK Integration Test",
+  "homepage": "https://developer.commercecloud.com/s/",
+  "scripts": {
+    "depcheck": "depcheck",
+    "lint": "eslint \"**/*.ts\" --quiet --fix",
+    "pretest": "npm run lint && npm run depcheck",
+    "test": "nyc mocha \"tests/*.test.*\"",
+    "fetch:internal": "npm pack ../packages/core && npm pack ../packages/generator && npm install ",
```

Comment:
Does this drop the packages into the current working directory?

Reply:
Also why fetch?

Reply:
> Does this drop the packages into the current working directory?
@monkey-jeff Yes it does, is that an issue?

Reply:
> Also why fetch?

@monkey-jeff because it's fetching the SDK and pulling it into the integration test package...

Reply:
I would rename it since in javascript land fetch is the primary used http request library.  Where the "fetch api" has become a standard.

Reply:
how about pack:sdk?

Reply:
renamed to bundle:sdk